imports/core/ui/components/ProposalModal.vue

```diff
@@ -81,7 +90,10 @@
         if (this.signature) {
           this.$store.dispatch('voting/vote', this.signature);
```

Comment:
computed is used to return computed value, but as i understand right, you are using that to call the method if the property gets reactively changed? its better do that in the autorun. Vue has a `created()` callback (much like blaze), where `this.$autorun(()=>{})` is provided by a vue-meteor package. please move it there